### DISCUSSION TITLE
How to sync at a specific location?

### DISCUSSION THREAD

**Discussion by nazar2sfive** [abraunegg/onedrive#1501] [Q&A] [answered]
I dual boot my pc.
/dev/sda my main disk

/dev/sda2 = windows 10
/dev/sda3 = linux
/dev/sda4 = ntfs partition(my files including onedrive)

![image](https://user-images.githubusercontent.com/12841320/120477563-a17c7e00-c3cb-11eb-8c23-f4cebe68d5e4.png)

![image](https://user-images.githubusercontent.com/12841320/120477576-a6d9c880-c3cb-11eb-977b-20c9bced2b11.png)

I have disabled files on demand option. so i can access them locally on both linux and windows.

now to the question...
How can i use onedrive in linux to sync to that specific location ? Will this avoid downloading again those 71Gb of data and sync only the changed files?

**Comment by abraunegg** [comment, marked as answer]
@nazar2sfive 
> How can i use onedrive in linux to sync to that specific location ?

Please read the usage documentation, specifically in regards to the 'sync_dir' configuration option.

Links:
* https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md
